### PR TITLE
abbr changes

### PR DESCRIPTION
**Pull request by davebytemfg**

### CODE REVIEW

File: src/epub/text/bullet-with-his-name.xhtml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 			<p>“Yes to that too.”</p>
 			<p>“All right, then, let’s get started. This isn’t the only solar system we have to visit on this circuit.”</p>
 			<hr/>
-			<p>Ernie Meeker⁠—of Chicago, Illinois, <abbr>U.S.</abbr> of <abbr>A.</abbr>, Occident, Terra, Sol, Starswarm 37, Rim Sector, Milky Way Galaxy⁠—rubbed his chin and slanted across the street to a drugstore.</p>
```
Comment (davebytemfg) [inline src/epub/text/bullet-with-his-name.xhtml:33]: @vr8hub - not excited about what I did here? Any better suggestions for "U.S of A." I'm assuming if I wrap the whole thing in a abbr that lint will complain.

**Comment by vr8hub** [comment]
I had a similar situation in O. Henry, where a railroad name was something like "U. of A." or "I. and G. N.". In those cases, Alex wanted tags around each letter, because each letter was an abbreviation, but the "of" wasn't. In those cases, they didn't need to be initialisms since it was just a single letter.

This case is slightly different since U.S. is an abbreviation in its own right. So I would think the right thing to do here is `<abbr class="initialism">U. S.</abbr> of <abbr>A.</abbr>`.

**Comment by vr8hub** [comment]
Per Alex just now on the list, DC4 would be an initialism, e.g. `<abbr class="initialism">D.C.4</abbr>`. Lint is going to complain about that with a t-030; you can ignore it for now.

**Comment by davebytemfg** [comment]
And for 3D I'm going to do this: ``<abbr class="initialism">3D.</abbr>``
   or since 3 isn't an abbreviation, then it would be?
``<abbr class="initialism">3.D.</abbr>``

I'm guessing that why no period at the end of DC4?

**Comment by vr8hub** [comment]
Right, 4 isn't an abbreviation, it's a number. And I've asked about 3D; if it were me, I'd exception it and leave off the period, like we do for TV, but I'm waiting to hear back from Alex.

**Comment by davebytemfg** [comment]
Adding this to the example pile for alex, this is more of a serial number though??

```
<p>“Here’s my newest model spaceship, a <abbr class="initialism">DS-57-B.</abbr> Notice the detail.” The oldest boy shoved one of the submarine affairs in his face.</p>
```

(I found a better regex to find UPPERCASE strings and have more work to do to add more abbrs.

**Comment by vr8hub** [comment]
I agree, I wouldn't treat that as an abbreviation.

**Comment by vr8hub** [comment]
Did you see my last comment? That serial # isn't an abbreviation, it doesn't need a tag.

**Comment by davebytemfg** [comment]
Heh, yes, right after I pushed, just reverted that. :)

**Comment by davebytemfg** [comment]
@vr8hub I'm going to merge the various branches up into the main trunk this evening and restart the reviews from there. I THINK everything is completed, pending any outstanding answers from you and alex, but it will be better to apply those to a complete set of changes going forward. And then I can re-review everything in one build.

**Comment by vr8hub** [comment]
Agreed.

**Comment by vr8hub** [comment]
Sorry, Dave, we've had a change—Alex did some investigation and DC4 is more like your DS-57-B, i.e. it doesn't appear the "DC" stands for anything. So it doesn't need an `abbr` tag at all.

**Comment by vr8hub** [comment]
And we have a ruling on [2-4]D as well; those are now exceptions like TV, so they're initialisms, but should not have periods.

**Comment by vr8hub** [comment]
Dang. Sorry, Dave, I didn't catch that DC4 should be changed to DC-4. So sorry! Alex said that's the way Boeing lists it on their web site.

**Pull request merged**